gallery/psget/module/psget_update-module.md

```diff
@@ -79,8 +79,36 @@ Version Name Repository Description
 2.8.1 ContosoServer MSPSGallery ContosoServer module
 ```
 
+### Update the module with a prerelease version, requires -AllowPrerelease flag
+```powershell
+PS C:\\windows\\system32> Get-InstalledModule
+Version Name Repository Description
+------- ---- ---------- -----------
+1.0 ContosoServer MSPSGallery ContosoServer module
+1.5 ContosoServer MSPSGallery ContosoServer module
+2.0 ContosoServer MSPSGallery ContosoServer module
+2.8.1 ContosoServer MSPSGallery ContosoServer module
+
+PS C:\\windows\\system32> Find-Module ContosoServer -AllowPrerelease
```

Comment:
Don't need the double-slash (\\) inside of the code fenced block.

Reply:
Addressed in update